packages/lascar/lascar.0.5/opam

```diff
@@ -1,14 +1,12 @@
 opam-version: "2.0"
-name: "lascar"
-version: "0.5"
+synopsis: "A library for manipulating Labeled Transition Systems in OCaml"
 maintainer: "Jocelyn Sérot <jocelyn.serot@uca.fr>"
-authors: [
-  "Jocelyn Sérot <jocelyn.serot@uca.fr>"
-]
-homepage: "http://cloud.ip.univ-bpclermont.fr/~serot/lascar/"
+authors: "Jocelyn Sérot <jocelyn.serot@uca.fr>"
 license: "MIT"
-dev-repo: "git+https://github.com/jserot/lascar.git"
+homepage: "http://cloud.ip.univ-bpclermont.fr/~serot/lascar/"
 bug-reports: "jocelyn.serot@uca.fr"
+dev-repo: "git+https://github.com/jserot/lascar.git"
+depends: [ "camlp4" "ocamlfind" {build} ]
```

Comment:
the previous depends section was more expressive (lower bound on ocaml) and more readable (indentation). Could you put it back?